src/app/screens/Enable/index.tsx

```diff
@@ -26,13 +27,21 @@ function Enable(props: Props) {
   const { t: tCommon } = useTranslation("common");
 
   const enable = useCallback(() => {
-    setEnabled(true);
-    msg.reply({
-      enabled: true,
-      remember,
-      budget,
-    });
-  }, [budget, remember]);
+    try {
+      setLoading(true);
+      msg.reply({
+        enabled: true,
+        remember,
+        budget,
+      });
+    } catch (e) {
+      console.error(e);
+      if (e instanceof Error) toast.error(`${tCommon("error")}: ${e.message}`);
+    } finally {
+      setEnabled(true);
```

Comment:
What is `setEnabled` doing in this component? Looks like it's not being used?

Reply:
Nope, it isn't being used; even loading wasn't being used anywhere (wanted to ask if there was anything to be implemented for which they were used as placeholders)

Reply:
Might have a been draft code for the future which was partly removed because it wasn't used. Shall we remove it to avoid confusion?